workers/Dockerfile.tests

```diff
@@ -5,6 +5,8 @@ COPY common/dist/data-refinery-common-* common/
 # Get the latest version from the dist directory.
 RUN pip3 install common/$(ls common -1 | sort --version-sort | tail -1)
 
+RUN pip3 install pandas
```

Comment:
So if I'm reading this correctly, this is being installed here instead of the requirements file because it is only used for the test? Is there a separate requirements file for the test environment?

Reply:
Thus far this is the only requirement difference between test and non/test so it seems to make more sense to just do it in this one off manner rather than adding a whole new requirements file. I should leave a comment here explaining that though.

Reply:
Sure, putting a comment here would be useful

Reply:
I said I'd do it and then I forgot  to :(